app/src/main/java/bluedot/com/au/bluedotbrazeintegrationapp/MainActivity.kt

```diff
@@ -1,18 +1,101 @@
 package bluedot.com.au.bluedotbrazeintegrationapp
 
+import android.app.Notification
+import android.app.NotificationChannel
+import android.app.NotificationManager
+import android.content.Context
+import android.graphics.Color
+import android.os.Build
 import android.os.Bundle
-import android.widget.Button
 import androidx.appcompat.app.AppCompatActivity
-import au.com.bluedot.point.net.engine.ServiceManager
+import androidx.core.app.NotificationCompat
+import au.com.bluedot.point.net.engine.GeoTriggeringService
+import bluedot.com.au.bluedotbrazeintegrationapp.databinding.ActivityMainBinding
 
 class MainActivity : AppCompatActivity() {
+    private lateinit var binding: ActivityMainBinding
+
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
-        setContentView(R.layout.activity_main)
+        binding = ActivityMainBinding.inflate(layoutInflater)
+        val view = binding.root
+
+        setContentView(view)
+
+        binding.bStartStopSDK.setOnClickListener {
+            val app: MainApplication = this.application as MainApplication
+
+            if (GeoTriggeringService.isRunning()) {
+                binding.bStartStopSDK.text = getString(R.string.stop_sdk)
+                GeoTriggeringService.stop(this, app)
+            } else {
+                binding.bStartStopSDK.text = getString(R.string.start_sdk)
+
+                val notification: Notification = createNotification()
 
-        findViewById<Button>(R.id.bStopSDK).setOnClickListener {
-            ServiceManager.getInstance(this).stopPointService()
+                GeoTriggeringService.builder()
+                    .notification(notification)
+                    .start(applicationContext, app)
+            }
             finish()
```

Comment:
Call to finish() was useful when SDK was stopped but now as Georigger is started here activity should not be finished, activity should be displayed to the user